src/main/resources/templates/layout.html

```diff
@@ -40,5 +44,22 @@
     <div class="content-container">
         <div layout:fragment="content"></div>
     </div>
+
+
+    <footer>
+        <div class="footer-container">
+            <div>WhaleSpotting</div>
+            <div><i class="fa fa-github"></i><a  href="https://github.com/charlottemothersole/WhaleSpotting-Java"> Github repository </a></div>
+            <div>©Copyright <span id="currentYear"></span>, BootCamp Project Amex</div>
+            <script>
+                    console.log("here");
+                    const currentDate = new Date();
+                    let currentYear = currentDate.getFullYear();
+                    document.getElementById("currentYear").innerHTML = currentYear;
+            </script>
+        </div>
+
+    </footer>
```

Comment:
You might not need the extra div in here, you _might_ just be able to do the below.

Additionally, I'd call what you have in the footer more like `span`s than `div`s
```suggestion
    <footer class="footer-container">
        <span>WhaleSpotting</span>
        <span><i class="fa fa-github"></i><a  href="https://github.com/charlottemothersole/WhaleSpotting-Java"> Github repository </a></span>
        <span>©Copyright <span id="currentYear"></span>, BootCamp Project Amex</span>
        <script>
                const currentDate = new Date();
                let currentYear = currentDate.getFullYear();
                document.getElementById("currentYear").innerHTML = currentYear;
        </script>
    </footer>
```

Reply:
The divs help with the flex box to allow the footer being responsive.